modules/twinklei18n.js

```diff
@@ -0,0 +1,53 @@
+TwinkleMessages = {
+	'en': {
+		'tw-core-portlet-name': 'TW',
+		'tw-core-summary': ' ([[WP:TW|TW]])',
+		'tw-core-summary-del': ' ([[WP:TW|TW]])',
+		'tw-core-summary-prot': ' ([[WP:TW|TW]])',
+		'tw-core-script-name': 'Twinkle',
+		'tw-core-help': 'Twinkle Help',
+		'tw-core-help-page': 'WP:TW/DOC#shared',
+		'tw-morebits-info': 'Info',
+		'tw-sharedip-label': 'Shared IP',
```

Comment:
Will do as advised. I will also add a 'qqq' group of translations to explain what the messages are when necessary. This can be then used on TranslateWiki to help boost TW translations!